avocado/core/parser.py

```diff
@@ -46,6 +51,32 @@ def _get_option_tuples(self, option_string):
         return []
 
 
+class StdoutAlreadyClaimedError(Exception):
+
+    """
+    Signals that other player already claimed exclusive rights on STDOUT
+    """
+
+
+class FileOrStdoutAction(argparse.Action):
```

Comment:
That would be less natural (Pythonic) than using an Action.  The whole point of providing `FileOrStdoutAction` is that plugin writers can use it as just another action when they add command line options.  I fail to see how requiring a user to call some parser internal funcional would be simpler.

Also the action is `__call__()`ed when the argument are parsed, that is, by `argparse.ArgumentParser.parse_args()`, so introducing something like `avocado.core.parser.claim_stdout` is more cumbersome.

As a side note, the trend that I'd like to see, is to move away from our custom objects that add little value, such as `avocado.core.parser.Parser`, and just make them extended (as needed) versions of standard library code, such as `argparse.ArgumentParser`.  Because of that, I favour the more Pythonic way of using an Action.